src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/AbstractAuditLog.java

```diff
@@ -80,33 +79,15 @@
 import com.google.common.io.BaseEncoding;
 
 public abstract class AbstractAuditLog implements AuditLog {
+    protected final Logger logger = LogManager.getLogger(this.getClass());
```

Comment:
Sorry if I missed that rename in the prior reviews. Can you please rename it back to `log` as other classes mostly use `log` for the `Logger`.

Reply:
`logger` seems more appropriate than `log`

Reply:
I don't have a strong preference of `log` over `logger`, just want to be consistent. For me, both `log` and `logger` are fine.

As the majority of other classes use `log` and the change is not part of `AuditConfig` refactoring, let's keep it as `log` in this PR/commit.